src/_conversions.scss

```diff
@@ -1,3 +1,5 @@
+@use "sass:math";
```

Comment:
I'm surprised this passes prettier. I'll want to check that out.

Reply:
Because of the double quotes?

If so, it looks like [Sass's own documentation](https://sass-lang.com/documentation/at-rules/use) is inconsistent / doesn't care about double ("src/corners") vs. single ('foundation/code'), and for Prettier we'd need to use the [Quotes option](https://prettier.io/docs/en/options.html#quotes).

Unrelated, I guess maybe if one wants to be a good citizen alphabetizing the modules would be proper.

Reply:
Looks like it's a bug to me https://github.com/prettier/prettier/issues/11053